dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoryHelper.java

```diff
@@ -61,6 +66,29 @@ public CategoryView toCategoryView(final Category category, final User user) {
                 .sortOrder(category.getSortOrder())
                 .categoryVelocityVarName(category.getCategoryVelocityVarName())
                 .build();
+
+
+        return Response.ok(new ResponseEntityView(categoryView)).build();
```

Comment:
you need to use a sub class if use Response as a return type, so swagger may do the doc